wdae/wdae/pheno_tool_api/tests/test_pheno_tool_api.py

```diff
@@ -20,7 +20,6 @@
 
 
 @pytest.mark.parametrize("url,method,body", [
-    (TOOL_URL, "post", QUERY),
     (TOOL_DOWNLOAD_URL, "post", QUERY),
```

Comment:
This test parametrization should be updated to support different expected response status codes, as this test is intended to test permissions